README.md

```diff
@@ -73,6 +73,26 @@ marketplace](https://marketplace.visualstudio.com/items?itemName=DigitalAssetHol
 
 You can follow the [instructions](https://github.com/moodmosaic/ide-haskell-ghcide#readme) to install with `apm`.
 
+### Using with Sublime Text
+
+* Install [LSP](https://packagecontrol.io/packages/LSP)
+* Press Ctrl+Shift+P or Cmd+Shift+P in Sublime Text and search for *Preferences: LSP Settings*, then paste these settings
+```
+{
+  "clients":
+  {
+    "haskell-ide-engine":
```

Comment:
```suggestion
    "ghcide":
```
Just to avoid confusion. Unless there's a reason it should be called that? (I know little about sublime text).

Reply:
Very good catch! For a second I thought it had to match the name in [here](https://microsoft.github.io/language-server-protocol/implementors/servers/) but I just renamed it to `ghcide` and it worked fine. (Even restarted Sublime Text to make sure.) Thank you :100:  :+1:

---

And looks better now as it says *ghcide* initialized, the very first time it'll invoke ghcide:

![image](https://user-images.githubusercontent.com/287532/71213620-45e03280-22b4-11ea-9de7-584b0f34171a.png)